### PR TITLE
[14.0][ADD] stock_service_level

### PR DESCRIPTION
**Pull request by petrus-v**
This is draft just exposing my plan in the description.rst ! Please Feel free to comment suggest other solutions, name or repo to host those sources or any other comments.

### CODE REVIEW

File: stock_service_level/readme/DESCRIPTION.rst
Comment (alexandregaldeano) [inline stock_service_level/readme/DESCRIPTION.rst]:
```suggestion
Implementation roadmap in different modules:
```

File: stock_service_level/readme/DESCRIPTION.rst
Comment (alexandregaldeano) [inline stock_service_level/readme/DESCRIPTION.rst]:
```suggestion
* [ ] link `stock.move` to service level model
```

**Comment by petrus-v** [comment]
note to myself to investigate:

Why gh actions appears to be testing with an old unmaintained python version (is it the current debian version when odoo 14.0 was released first)... this make unavailable some new features like walrus operators which are currently supported by the oldest supported python version: 3.8 !!

As far I remember I've used it on an other repository without error, maybe a simple copier update would fix that !

File: stock_service_level/__manifest__.py
Comment (barkat-matthias) [inline stock_service_level/__manifest__.py]: Memo to change the version to 14.0.1.0.0

**Review by barkat-matthias — APPROVED** [review]

File: stock_service_level/models/stock_service_level.py
Comment (rousseldenis) [inline stock_service_level/models/stock_service_level.py]:
```suggestion
            "Code must be unique.",
```

prefer explicit wording

File: stock_service_level/models/stock_move.py
Comment (rousseldenis) [inline stock_service_level/models/stock_move.py:10]:
```suggestion
    service_level_id = fields.Many2one(
        index="btree_not_null",
```

File: stock_service_level/models/stock_service_level.py
Comment (rousseldenis) [inline stock_service_level/models/stock_service_level.py]:
```suggestion
    code = fields.Char(required=True, index=True)
```

**Review by rousseldenis — APPROVED** [review]
Code review. Minor comments

**Comment by petrus-v** [comment]
I declare myself as maintainer but this is a new module... sorry for this try!

**Comment by petrus-v** [comment]
Hi @rousseldenis is there any thing missing before merging here ?

Thanks for your time

**Comment by rousseldenis** [comment]
@petrus-v Could you squash your commits or improve the second one's message ?

**Comment by petrus-v** [comment]
Thanks @rousseldenis,  commits squashed ! CI is Green :fireworks:

**Comment by rousseldenis** [comment]
Comments have been attended.

/ocabot merge patch

**Pull request merged**
